src/modules/fancyzones/FancyZonesLib/WorkArea.cpp

```diff
@@ -297,7 +298,7 @@ IFACEMETHODIMP WorkArea::MoveSizeEnd(HWND window, POINT const& ptScreen) noexcep
             SaveWindowProcessToZoneIndex(window);
         }
     }
-    Trace::WorkArea::MoveSizeEnd(m_activeZoneSet);
+    Trace::WorkArea::MoveOrResizeEnd(m_activeZoneSet);
```

Comment:
Should this be MoveOrResizeEnded? I see "Ended" is used in other spots on the PR, but not sure if it matters here.

Reply:
Also, what exactly triggers a MoveOrResize event? And why are move & resize grouped together instead of as separate events? Does it have to do with how this is triggered?

Reply:
> Should this be MoveOrResizeEnded? I see "Ended" is used in other spots on the PR, but not sure if it matters here.

The Event name passed through telemetry is "FancyZones_MoveOrResizeEnded".

Reply:
> Also, what exactly triggers a MoveOrResize event? And why are move & resize grouped together instead of as separate events? Does it have to do with how this is triggered?

This event happens when we're dragging a Window to set up the zone (or changing monitors with the Window dragged). From the name it seems to imply that it's also used for Resizing.
@SeraphimaZ , correct me here if I'm wrong.

Reply:
Yes, this event is triggered when a window is moved or resized. The reason for the naming and grouping is the system event name we handle, see `EVENT_SYSTEM_MOVESIZEEND` https://docs.microsoft.com/en-us/windows/win32/winauto/event-constants

Reply:
I see, so does MoveOrResize trigger even if the window isn't being used with FancyZones? I worry about this causing noise.

Reply:
> does MoveOrResize trigger even if the window isn't being used with FancyZones?

By this step events are filtered already, so there would be events only from windows that are being used with FZ.